gateway/api/services/file_storage.py

```diff
@@ -0,0 +1,116 @@
+"""
+This file stores the logic to manage the access to data stores
+"""
+import glob
+import logging
+import os
+
+from django.conf import settings
+
+from utils import sanitize_file_path
+
+USER_STORAGE = "user"
+PROVIDER_STORAGE = "provider"
+
+SUPPORTED_FILE_EXTENSIONS = [".tar", ".h5"]
+
+logger = logging.getLogger("gateway")
+
+
+class FileStorage:  # pylint: disable=too-few-public-methods
+    """
+    The main objective of this class is to manage the access of the users to their storage.
+
+    Attributes:
+        username (str): storgae user's username
+        file_path (str): base user storage file path
+    """
+
+    def __init__(
+        self,
+        username: str,
+        working_dir: str,
```

Comment:
Oh yes, I remember now. Enums with strings are only supported with `StrEnum` only available in Python 3.11 that's the reason because I decided not to use them at the end.

Reply:
Issue solved in [4eb90b9](https://github.com/Qiskit/qiskit-serverless/pull/1529/commits/4eb90b9ffd74ed4b1ba4ba9a30ad1dbacc91cd9a)